source/includes/_pis.md

```diff
@@ -125,6 +125,143 @@ At the moment, we don't support the `payment-details` endpoint.
+| EUR                | `UK.OBIE.SEPACreditTransfer` | `UK.OBIE.IBAN`             | IBAN                          | -                                       | -                         | -                            | -                      |                                                                              |
+| EUR                | `UK.OBIE.SWIFT`              | `UK.OBIE.IBAN`             | IBAN                          | -                                       | `UK.OBIE.BICFI`           | BIC                          | -                      |                                                                              |
+| RON                | -                            | `UK.OBIE.IBAN`             | IBAN                          | -                                       | `UK.OBIE.BICFI`           | BIC                          | Required               |                                                                              |
+| USD                | `UK.MONZO.ABA`               | `UK.OBIE.BBAN`             | Account Number                | -                                       | `UK.OBIE.NCC.US`          | ABA Routing Number           | Required               |                                                                              |
+| USD                | `UK.MONZO.FEDWIRE`           | `UK.OBIE.BBAN`             | Account Number                | -                                       | `UK.OBIE.NCC.US`          | Fedwire Routing Number       | Required               |                                                                              |
+| USD                | `UK.OBIE.SWIFT`              | `UK.OBIE.BBAN`             | Account Number                | -                                       | `UK.OBIE.BICFI`           | BIC                          | Required               |                                                                              |
+
+`CreditorAccount.Name` is required for all currencies.
+
+`Initiation.DestinationCountryCode` is required for all currencies.
+
+If `Creditor.PostalAddress` is required, the following fields must be provided:
+
+  * `TownName`
+  * `PostCode`
+  * Either `AddressLine` or `BuildingNumber` + `StreetName`
+
+Risk.BeneficiaryAccountType is required and supports the following values:
+  * `Personal`
+  * `JointPersonal`
+  * `PersonalSavingsAccount`
+  * `Business`
+  * `BusinessSavingsAccount`
+
+For USD payments, the account type (`Checking` or `Savings`) is determined based on `Risk.BeneficiaryAccountType`.
+
+<aside class="notice">
+If there are insufficient funds in the account, authorisation will fail and an error will be returned on redirection with the code `access_denied` and `error_description` being "Insufficient funds in selected account to make requested payment."
```

Comment:
nit: have we done this yet?

Reply:
Yes, it is done - as we inherited the logic from domestic payments 👍